hpccm/building_blocks/mkl.py

```diff
@@ -120,7 +120,7 @@ def __instructions(self):
         self += packages(
             _apt_key=True,
             apt_keys=['https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-{}.PUB'.format(self.__year)],
-            apt_repositories=['deb https://apt.repos.intel.com/mkl all main'],
+            apt_repositories=['deb [signed-by=/usr/share/keyrings/'+'GPG-PUB-KEY-INTEL-SW-PRODUCTS-{}.gpg'.format(self.__year)+'] https://apt.repos.intel.com/mkl all main'],
```

Comment:
If would be clearer to put the format at the end of the whole string, i.e.,
```
apt_repositories=['deb [signed-by=/usr/share/keyrings/GPG-PUB-KEY-INTEL-SW-PRODUCTS-{}.gpg] https://apt.repos.intel.com/mkl all main'.format(self.__year)],
```

Reply:
Yes, indeed.